lib/Swagger/src/SiteAttribute/Api/SiteAttributeApi.php

```diff
@@ -0,0 +1,535 @@
+{
+
+    /**
+     * API Client
+     *
+     * @var \Swagger\Client\ApiClient instance of the ApiClient
+     */
+    protected $apiClient;
+
+    /**
+     * Constructor
+     *
+     * @param \Swagger\Client\ApiClient|null $apiClient The api client to use
+     */
+    public function __construct(\Swagger\Client\ApiClient $apiClient = null)
+    {
+        if ($apiClient == null) {
+            $apiClient = new ApiClient();
+            $apiClient->getConfig()->setHost('https://localhost/site-attribute');
+        }
+
+        $this->apiClient = $apiClient;
+    }
+
+    /**
+     * Get API client
+     *
+     * @return \Swagger\Client\ApiClient get the API client
+     */
+    public function getApiClient()
+    {
+        return $this->apiClient;
+    }
+
+    /**
+     * Set the API client
+     *
+     * @param \Swagger\Client\ApiClient $apiClient set the API client
+     *
+     * @return SiteAttributeApi
+     */
+    public function setApiClient(\Swagger\Client\ApiClient $apiClient)
+    {
+        $this->apiClient = $apiClient;
+        return $this;
+    }
+
+    /**
+     * Operation deleteAttribute
+     *
+     * Deletes attribute for this site
+     *
+     * @param int $site_id id of the site to be modified (required)
+     * @param string $attr_name Name of attribute to be deleted for specified site (required)
+     * @return void
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function deleteAttribute($site_id, $attr_name)
+    {
+        list($response) = $this->deleteAttributeWithHttpInfo($site_id, $attr_name);
+        return $response;
+    }
+
+    /**
+     * Operation deleteAttributeWithHttpInfo
+     *
+     * Deletes attribute for this site
+     *
+     * @param int $site_id id of the site to be modified (required)
+     * @param string $attr_name Name of attribute to be deleted for specified site (required)
+     * @return array of null, HTTP status code, HTTP response headers (array of strings)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function deleteAttributeWithHttpInfo($site_id, $attr_name)
+    {
+        // verify the required parameter 'site_id' is set
+        if ($site_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $site_id when calling deleteAttribute');
+        }
+        // verify the required parameter 'attr_name' is set
+        if ($attr_name === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $attr_name when calling deleteAttribute');
+        }
+        // parse inputs
+        $resourcePath = "/site/{siteId}/attr/{attrName}";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('*_/_*'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Accept" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)